docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -135,19 +132,6 @@ def __build_http_proxy_image(self):
 
         return self.__build_image(dockerfile)
 
-    def __build_nifi_image(self):
-        dockerfile = dedent(r"""\
-                FROM {base_image}
-                USER root
-                RUN sed -i -e 's/^\(nifi.remote.input.host\)=.*/\1={name}/' {nifi_root}/conf/nifi.properties
-                RUN sed -i -e 's/^\(nifi.remote.input.socket.port\)=.*/\1=5000/' {nifi_root}/conf/nifi.properties
```

Comment:
is `nifi.remote.input.socket.port` no longer needed?

Reply:
I don't think it was even used before this change, as port 8080 was used for site to site tests.